app/Importers/BirdDataImporter.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Importers;
+
+use App\Models\City;
+use App\Models\CityAlternativeName;
+use App\Models\Country;
+use App\Services\MapboxGeocodingService;
+use Throwable;
+
+class BirdDataImporter extends DataImporter
+{
+    private const PROVIDER_ID = 1;
+
+    protected string $fetchedData;
+
+    public function extract(): static
+    {
+        try {
+            $html = file_get_contents("https://www.bird.co/map/");
+        } catch (Throwable) {
+            $this->createImportInfoDetails("400", self::PROVIDER_ID);
```

Comment:
Is `400` status code?

Reply:
If it is HTTP status code, better and more readable should be that class and codes in this class - `Symfony\Component\HttpFoundation\Response`

```suggestion
            $this->createImportInfoDetails(Response::HTTP_BAD_REQUEST, self::PROVIDER_ID);
```

Reply:
It's status code pretty similar to the http status code, but actual meaning is explained in table `codes`